packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1260,7 +1260,15 @@ class _DirectionallyExtendCaretSelectionAction<T extends DirectionalCaretMovemen
   }
 }
 
-class _SelectableRegionContainerDelegate extends MultiSelectableSelectionContainerDelegate {
+/// A delegate for the [SelectionContainer]s of [SelectableRegion]s and their
+/// children.
+class SelectableRegionContainerDelegate extends MultiSelectableSelectionContainerDelegate {
```

Comment:
We need a more generic name if we open this up. Maybe call StaticMultiSelectableSelectionContainerDelegate.